float64 thrust
float64 torque_phi
float64 torque_theta
float64 torque_psi
